spark/unitycatalog/src/test/java/io/sparkuctest/UnityCatalogSupportSuite.java

```diff
@@ -54,8 +54,8 @@ public class UnityCatalogSupportSuite extends UnityCatalogSupport {
   private SparkSession spark;
 
   @BeforeEach
-  public void setUp() throws Exception {
-    super.setup(); // Start UC server first
+  public void setUpSparkSession() throws Exception {
```

Comment:
Do we still need this class, as we've already had the `UCDeltaTableIntegrationBaseTest`.